snowpack/src/commands/install.ts

```diff
@@ -398,13 +399,15 @@ ${colors.dim(
   };
   if (Object.keys(installEntrypoints).length > 0) {
     try {
+      logger.debug(`running installer with options: ${util.format(inputOptions)}`);
       const packageBundle = await rollup(inputOptions);
       logger.debug(
         `installing npm packages:\n    ${Object.keys(installEntrypoints).join('\n    ')}`,
       );
       if (isFatalWarningFound) {
         throw new Error(FAILED_INSTALL_MESSAGE);
       }
+      logger.debug(`writing install results to disk`);
```

Comment:
reasoning: outputOptions is fairly static, this mainly serves to just separate a bug that happens during `rollup()` from one that happens during `bundle.write()`.